<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="serial_widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>751</width>
      <height>291</height>
     </rect>
    </property>
    <widget class="QPushButton" name="thlButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>温度湿度亮度</string>
     </property>
    </widget>
    <widget class="QWidget" name="Motor_widget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>251</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/background.jpg);</string>
     </property>
     <widget class="QFrame" name="Motor_frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>201</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="forwardButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>放下</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reversalButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>收起</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stopButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="Motor_Button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>遮光罩</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LED_Button">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>LED</string>
     </property>
    </widget>
    <widget class="QWidget" name="LED_widget" native="true">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>171</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/background.jpg);</string>
     </property>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>201</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="LED4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>门口</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LED3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>管理室</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LED1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>管理室</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LED2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>卫生间</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PWM_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>PWM</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="thl_widget" native="true">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>70</y>
       <width>261</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/background.jpg);</string>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLCDNumber" name="lcd_temp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>64</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcd_humi">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>64</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcd_light">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>64</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>34</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>温度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>34</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>湿度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>34</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>亮度</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="ray_label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLCDNumber" name="label_ultra">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>26</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>红外</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>39</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>超声波</string>
      </property>
     </widget>
     <zorder>frame_2</zorder>
     <zorder>label_6</zorder>
     <zorder>label_6</zorder>
     <zorder>ray_label</zorder>
     <zorder>label_ultra</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
    </widget>
    <widget class="QPushButton" name="IntelButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>92</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>个人控制</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="comButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>6</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>16</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>开关</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="hour_lcd">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="testlcd">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ComBrowser">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
